src/test/java/uk/gov/pay/api/service/GetPaymentServiceTest.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.pay.api.service;
+
+import au.com.dius.pact.consumer.PactVerification;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.pay.api.app.RestClientFactory;
+import uk.gov.pay.api.app.config.PublicApiConfig;
+import uk.gov.pay.api.app.config.RestClientConfig;
+import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.model.CardPayment;
+import uk.gov.pay.api.model.PaymentState;
+import uk.gov.pay.api.model.TokenPaymentType;
+import uk.gov.pay.api.model.links.PaymentWithAllLinks;
+import uk.gov.pay.api.model.links.PostLink;
+import uk.gov.pay.commons.model.SupportedLanguage;
+import uk.gov.pay.commons.testing.pact.consumers.PactProviderRule;
+import uk.gov.pay.commons.testing.pact.consumers.Pacts;
+
+import javax.ws.rs.client.Client;
+import java.util.Collections;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GetPaymentServiceTest {
+
+    private static final String ACCOUNT_ID = "123456";
+    private static final String CHARGE_ID = "ch_123abc456def";
+
+    private GetPaymentService getPaymentService;
+
+    @Rule
+    public PactProviderRule connectorRule = new PactProviderRule("connector", this);
+
+    @Mock
+    private PublicApiConfig mockConfiguration;
+
+    @Before
+    public void setup() {
+        // We will actually send real requests here, which will be intercepted by pact
```

Comment:
I don't think we need this comment here as it's clear from line 48 we're returning a non-mock endpoint created by the PactProviderRule.